pkg/kubernetes/util.go

```diff
@@ -18,16 +19,16 @@ const (
 
 // GetHostnamesForService returns a list of hostnames over which the service can be accessed within the local cluster.
 // If 'sameNamespace' is set to true, then the shorthand hostnames service and service:port are also returned.
-func GetHostnamesForService(service *corev1.Service, sameNamespace bool) []string {
+func GetHostnamesForService(svc *corev1.Service, locality service.Locality) []string {
```

Comment:
Lets avoid the name change from service to svc

Reply:
it ends up conflicting with the service package